reports/qc/qc.Rmd

```diff
@@ -326,9 +346,11 @@ p_data %>%
   xlab("")
 ```
 
+Table listing all samples {nrow(all_data)} samples in {n_batches} batches.
```

Comment:
```suggestion
Table listing all {nrow(all_data)} samples in {n_batches} batches.
```

Reply:
Forgot to merge these suggestions! Sorry and thanks for fixing the typos.